oauthenticator/globus.py

```diff
@@ -210,6 +215,16 @@ def get_callback_url(self, handler=None):
                             )
         return self.oauth_callback_url
 
+    def logout_url(self, base_url):
+        return url_path_join(base_url, 'logout')
+
+    def get_handlers(self, app):
+        return [
```

Comment:
Perhaps

```python
return super().get_handlers(app) + [(r'/logout', self.logout_handler)]
```

to only add handlers, rather than duplicate

Reply:
Will do, thanks!